players.js

```diff
@@ -1,6 +1,7 @@
-const {Clutter,Gio,GLib,GObject,Shell,St} = imports.gi;
+const {Clutter,Gio,GLib,GObject,Shell,St,Meta} = imports.gi;
 const ExtensionUtils = imports.misc.extensionUtils;
 const CurrentExtension = ExtensionUtils.getCurrentExtension();
+const altTab = imports.ui.altTab;
```

Comment:
we aren't using this

Reply:
Yeah, same thing, I tried but couldn't figure out how to implement. It can be removed too.